elasticache/memcached/Acornfile

```diff
@@ -0,0 +1,121 @@
+	nodeType: "cache.t4g.micro"
+
+	// The number of cache nodes used in the elasticache cluster. Default value is 1.
+	numNodes: 1
+
+	// Enables TLS for connections to the cluster when true. Default value is false.
+	transitEncryption: false
+}
+
+services: admin: {
+	name:    "Memcached Admin"
+	default: true
+	generated: job: "apply"
+}
+
+jobs: apply: {
+	build: {
+		context:    "../"
+		dockerfile: "../memcached.Dockerfile"
+		additionalContexts: common: "../../libs"
+	}
+	//                         entrypoint: "sleep"
+	//                         command: ["7200"]
+	files: "/app/config.json": std.toJSON(args)
+	env: {
+		CDK_DEFAULT_ACCOUNT:          "@{secrets.aws-context.account-id}"
+		CDK_DEFAULT_REGION:           "@{secrets.aws-context.aws-region}"
+		VPC_ID:                       "@{secrets.aws-context.vpc-id}"
+		ACORN_ACCOUNT:                "@{acorn.account}"
+		ACORN_NAME:                   "@{acorn.name}"
+		ACORN_PROJECT:                "@{acorn.project}"
+		ACORN_EXTERNAL_ID:            "@{acorn.externalID}"
+		CDK_RUNNER_DELETE_PROTECTION: "\(args.deletionProtection)"
+	}
+	events: ["create", "update", "delete"]
+	permissions: rules: [{
+		apiGroup: "aws.acorn.io"
+		verbs: [
+			"cloudformation:DescribeStacks",
+			"cloudformation:CreateChangeSet",
+			"cloudformation:DescribeStackEvents",
+			"cloudformation:DescribeStackResources",
+			"cloudformation:DescribeChangeSet",
+			"cloudformation:ExecuteChangeSet",
+			"cloudformation:PreviewStackUpdate",
+			"cloudformation:UpdateStack",
+			"cloudformation:GetTemplateSummary",
+			"cloudformation:DeleteStack",
+			"cloudformation:GetTemplate",
+			"ec2:*",
+			"elasticache:*",
+			"secretsmanager:*",
+		]
+		resources: ["*"]
+	}, {
+		apiGroup: "aws.acorn.io"
+		verbs: [
+			"iam:CreateServiceLinkedRole",
+		]
+		resources: ["arn:aws:iam::*:role/aws-service-role/elasticache.amazonaws.com/AWSServiceRoleForElastiCache"]
+	}, {
+		apiGroup: "api.acorn.io"
+		verbs: [
+			"create",
+		]
+		resources: ["events"]
+	}]
+}
+
+secrets: admin: {
+	type: "generated"
+	params: job: "apply"
+}
+
+secrets: "aws-context": {
+	name:     "AWS Context"
+	external: "context://aws"
+	type:     "opaque"
+	data: {
+		"account-id": ""
+		"vpc-id":     ""
+		"aws-region": ""
+	}
+}
+
+localData: info: """
+	## How To Use ([examples](https://github.com/acorn-io/aws/tree/main/elasticache/memcached/examples))
+
+	1) Link your app with this acorn via an `external` service named "memcached".
+
+	```typescript
+			 services: memcached: {
+					external: "@{acorn.name}"
+			 }
+			 containers: app: {
+					build: context: "./"
+					ports: publish: ["8080/http"]
+					env: {
+							REDIS_HOST: "@{@{service.}memcached.address}"
+							REDIS_PORT: "@{@{service.}memcached.data.port}"
+							REDIS_PASSWORD: "@{@{service.}memcached.secrets.admin.token}"
```

Comment:
No tokens, right?

Reply:
No token. Just forgot to update this part of the Acornfile before. Fixed in latest commit.